manual/cops_lint.md

```diff
@@ -1520,7 +1520,7 @@ rather than meant to be part of the resulting symbols.
 
 Enabled by default | Safe | Supports autocorrection | VersionAdded | VersionChanged
 --- | --- | --- | --- | ---
-Enabled | Yes | No | 0.75 | -
+Pending | Yes | No | 0.81 | -
```

Comment:
This change is by `rake generate_cops_documentation` but why...?

Reply:
I messed up, but @koic just fixed my mistake. Just rebase on top of `master` and this will dissapper.

Reply:
Thanks for your response! I've fixed it!